stdlib/ballerina-http/src/main/java/org/ballerinalang/net/http/actions/httpclient/Delete.java

```diff
@@ -71,11 +69,7 @@ protected HTTPCarbonMessage createOutboundRequestMsg(Context context) {
         // Extract Argument values
         HTTPCarbonMessage cMsg = super.createOutboundRequestMsg(context);
         cMsg.setProperty(HttpConstants.HTTP_METHOD, HttpConstants.HTTP_METHOD_DELETE);
-
-        ObserverContext observerContext = ObservabilityUtils.getParentContext(context);
-        HttpUtil.injectHeaders(cMsg, ObservabilityUtils.getContextProperties(observerContext));
-        observerContext.addTags(HttpUtil.extractTags(cMsg));
-
+        HttpUtil.checkAndObserveHttpRequest(context, cMsg);
```

Comment:
Since this method call is common to all actions, We can move it to its super method. 
i.e createOutboundRequestMsg of AbstractHTTPAction class
Make sure change is done except for forward and execute actions. Because, in those actions, the super method is not invoked.

Reply:
Moved HttpUtil.checkAndObserveHttpRequest to AbstractHTTPAction.send method

Reply:
Make sure change is done except for forward and execute actions